server/app/auth/ProfileUtils.java

```diff
@@ -43,6 +45,22 @@ public Optional<CiviFormProfile> currentUserProfile(WebContext webContext) {
     return Optional.of(profileFactory.wrapProfileData(p.get()));
   }
 
+  public Optional<ApiKey> currentApiKey(Http.RequestHeader request) {
+    PlayWebContext webContext = new PlayWebContext(request);
+    return currentApiKey(webContext);
+  }
+
+  public Optional<ApiKey> currentApiKey(WebContext webContext) {
+    ProfileManager profileManager = new ProfileManager(webContext, sessionStore);
+    Optional<BasicUserProfile> p = profileManager.getProfile(BasicUserProfile.class);
```

Comment:
Suggestion: p -> profile

The public guide only addresses this for [param names](https://google.github.io/styleguide/javaguide.html#s5.2.6-parameter-names), but I think in general avoiding arbitrary abbreviations is best for readability.

Reply:
done